packages/build_package.js

```diff
@@ -64,6 +64,12 @@ const PACKAGES = {
     browsers: [],
     files: [...PLAYWRIGHT_CORE_FILES, ...FFMPEG_FILES, 'electron-types.d.ts'],
   },
+  'playwright-android': {
+    version: '0.0.2', // Manually manage playwright-android version.
+    description: 'A high-level API to automate Chrome for Android',
+    browsers: [],
+    files: [...PLAYWRIGHT_CORE_FILES, ...FFMPEG_FILES, 'android-types.d.ts', 'bin/android-driver.apk', 'bin/android-driver-target.apk'],
```

Comment:
Please take a look at npmignore file. I think we are either shipping apks to all packages, or ignoring them in playwright-android.